dotnet/samples/KernelSyntaxExamples/Example64_MultiplePromptTemplates.cs

```diff
@@ -3,7 +3,7 @@
 using System;
 using System.Threading.Tasks;
 using Microsoft.SemanticKernel;
-using Microsoft.SemanticKernel.TemplateEngine.Handlebars;
+using Microsoft.SemanticKernel.PromptTemplate.Handlebars;
```

Comment:
Should this be PromptTemplates (plural) rather than PromptTemplate singular?  Or PromptTemplating? Or just Templating?

Reply:
`PromptTemplate` is the term we use elsewhere in the code. This package contains support for the Handlebars prompt template format so the name seems reasonible.